app/containers/EpisodeAdminEditor/actions.js

```diff
@@ -5,11 +5,50 @@
  */
 
 import {
-  DEFAULT_ACTION,
+  UPDATE_EPISODE_PENDING,
+  UPDATE_EPISODE_SUCCESS,
+  UPDATE_EPISODE_FAILED,
+  DELETE_EPISODE_PENDING,
+  DELETE_EPISODE_SUCCESS,
+  DELETE_EPISODE_FAILED,
 } from './constants';
 
-export function defaultAction() {
+export function updateEpisodePending(episode) {
```

Comment:
These could probably be easier to read, i.e. `episodeUpdatePending`, `episodeDeletionSuccess` etc.